.github/workflows/ci.yml

```diff
@@ -0,0 +1,122 @@
+# Copyright(c) The Maintainers of Nanvix.
+# Licensed under the MIT License.
+
+name: "Nanvix CI/CD"
+
+on:
+  push:
+    branches: [dev]
```

Comment:
@ppenna this script will now (only) run on commits to `dev` branch, or PRs against `dev` with the status below (i.e. not in draft).

what other configurations would you like to enable?

Reply:
Let's run it on the same matching branches as before:

- dev
- enhancement-*
- feature-*
- bugfix-*

Reply:
Ok. Note that this will still run if someone opens a PR against `dev` that does not begin with `enhancement/feature/bugfix`. I will see if I can add that somehow.